### ISSUE TITLE
Create usable admin panel without dependencies

### ISSUE THREAD

**Issue by Nomafin** [devgeniem/redipress#17]
Update the RediPress admin panel to be more usable and so that it wouldn't use any 3rd party dependencies (like ACF and DustPress.js it does now).

**Comment by Nomafin** [comment]
Additional requirement:

Create a settings tab for the document indexing feature with following checkboxes that should be checked by default:

```PDF		application/pdf
Docx	application/vnd.openxmlformats-officedocument.wordprocessingml.document
Doc		application/msword
RTF		application/rtf
ODT		application/vnd.oasis.opendocument.text```

It should return an array of the values via `Admin::get`

**Comment by Nomafin** [comment]
Additional requirement:

Create a setting for fuzzy matching with possibilities to set it off or to distances of 1, 2 or 3.

**Comment by Nomafin** [comment]
Additional requirement:

Create a setting for post type weighting with own integer fields for all post types set to `1` by default.

**Comment by godbone** [comment]
The mime type listing should also include `application/pdf` for pdf and we could have the key be `mime_types`.

**Comment by Nomafin** [comment]
Implemented since 1.0.
